src/test/java/runner/TestFunctionalRunner.java

```diff
@@ -0,0 +1,19 @@
+package runner;
+
+import cucumber.api.CucumberOptions;
+import cucumber.api.junit.Cucumber;
+
+import org.junit.runner.RunWith;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(
+        features = "src/test/resources/features",
+        glue={"stepsDefinitions"},
+        tags = {"@Functional"},
+        plugin = {"com.vimalselvam.cucumber.listener.ExtentCucumberFormatter:target/cucumber-reports/report.html"},
+        monochrome = true
+)
+
+public class TestFunctionalRunner {
```

Comment:
No problem in get two runners, but we could get only one and specify the tag that you want to execute with mvn test -Dcucumber.options='--tags @Performance'